src/Event/Http/Psr7Bridge.php

```diff
@@ -157,4 +163,31 @@ private static function parseKeyAndInsertValueInArray(array &$array, string $key
 
         $pointer = $value;
     }
+    
+    /**
+     * Parse the username and password from the `Authorization` header.
+     * Only "Basic" is supported.
```

Comment:
```suggestion
     * Only "Basic" is supported for now.
```

Just to clarify that this isn't intentional and there is no reason to not change that later.